integration_tests/testsuite/test_logging_custom.py

```diff
@@ -50,13 +52,24 @@ def runTest(self):
 
             project_id = test_util.project_id()
 
-            FILTER = 'logName = ' \
-                     '"projects/{0}/logs/{1}" ' \
-                     'AND (textPayload:{2} OR jsonPayload.message:*) ' \
-                     'AND severity = "{3}"'.format(project_id,
-                                                   log_name,
-                                                   test_util.LOGGING_PREFIX,
-                                                   level)
+            environment = test_util.get_environment(self._base_url)
```

Comment:
Should we add this new endpoint to the integration test docs?

Reply:
yep thanks for the reminder, added